style.css

```diff
@@ -875,3 +880,510 @@ ul {
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    margin: 5% 0 0 5%;
+    border-radius: 24px;
+    background-color: #e5e5e5;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid #dfe1e6;
+    margin-top: 5%;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+    margin-bottom: 5%;
+  }
+
+  .card-container img {
+    width: 50%;
+    height: 100%;
+    border-radius: 8px;
+  }
+
+  .faha2 {
+    order: 2;
+  }
+
+  .work-fonony {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-left: 2%;
+    margin-left: 8rem;
+    width: 50%;
+    margin-top: 10rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .frame3-img {
+    display: flex;
+    align-items: center;
+  }
+
+  .project-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+    width: 60%;
+    font-size: 16px;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+    margin-left: -4rem;
+  }
+
+  ul.project-info li {
+    list-style-type: none;
+    position: relative;
+    white-space: nowrap;
+  }
+
+  ul.project-info li:not(:first-child)::before {
+    content: "\2022";
+    font-size: 40px;
+    position: absolute;
+  }
+
+  .fb {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #3b5998;
+  }
+
+  .full {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .year {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  #fb-title {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  #fb-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+    margin: 0;
+    width: 60%;
+    font-size: 16px;
+    margin-left: -3.5rem;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+  }
+
+  #back-list {
+    margin-left: -5.3rem;
+    margin-top: -9.5rem;
+  }
+
+  #fd-buttons {
+    margin-left: -5.4rem;
+  }
+
+  #description-fb {
+    margin-top: -2rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  .description {
+    display: flex;
+    margin-top: -3rem;
+    width: 100%;
+    margin-left: -5.3rem;
+  }
+
+  ul.languages-list li {
+    white-space: nowrap;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -9.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+    margin-top: 3rem;
+    background: #4053fc;
+    color: #fff;
+  }
+
+  /* About section */
+
+  .about-me {
+    display: flex;
+    flex-direction: row;
+    align-content: center;
```

Comment:
- [ ] Make sure the Languages, frameworks, and skills are well aligned to be exactly like the original design

> Your design
<img width="1188" alt="Screenshot 2023-05-04 at 17 19 02" src="https://user-images.githubusercontent.com/71140133/236268317-88eb84cf-cb19-4fcb-9bf9-fe5783385c0d.png">

> Figma design
<img width="571" alt="Screenshot 2023-05-04 at 17 18 44" src="https://user-images.githubusercontent.com/71140133/236268474-bc576b0d-e634-466d-a7f7-da101f51a847.png">